src/geowombat/core/stac.py

```diff
@@ -208,7 +216,7 @@ def _download_worker(item, extra: str, out_path: _Path) -> dict:
 
 
 def open_stac(

```

Comment:
Stack is still new to me but at some point we need to bring this into the tutorials in the docs.